bridge/bridgeapp/forms.py

```diff
@@ -1,38 +1,49 @@
 from django import forms
 from bridgeapp.models import Category
 
+# global variable with assigned loop to create a dictionary of category.id : category.type key:value pairs
 CAT_CHOICES = [(f"{category.id}", f"{category.type}")
                for category in Category.objects.all()]
 
 
 class ResponseForm(forms.Form):
     """Form used for Responses to Threads"""
+    # simple form with a single CharField
     body = forms.CharField(
         required=True,
         label='',
         widget=forms.Textarea(attrs={
             'rows': '3',
             'cols': '50',
+            # causes cursor to focus on text input area upon page load
             'autofocus': True,
+            # inserts placeholder text into text input area as prompt for user
             'placeholder': 'Write a response here...'
         }))
 
 
 class ThreadForm(forms.Form):
     """Form used for creation of Threads"""
     body = forms.CharField(
+        # requires entry in this field before form can be submitted
         required=True,
         label='',
         widget=forms.Textarea(attrs={
             'rows': '3',
             'cols': '50',
+            # causes cursor to focus on text input area upon page load
             'autofocus': True,
+            # inserts placeholder text into text input area as prompt for user
             'placeholder': 'Type a new question here then assign categories below...'
         }))
-
+    # assigns a multiple choice field to category_ids variable, allowing user to select categories for newly created thread
```

Comment:
For the reasons already stated in the above, I would use this comment for the entire field like so:
```
# multi-selection checkboxes that gathers `Category.id`s to submit with a new question
```
You can use thread insteadof question as you want.